models/reviewsModel.js

```diff
@@ -0,0 +1,23 @@
+const db = require("../db/connection.js");
+
+const fetchReviews = () => {
+  return db
+    .query(
+      `SELECT reviews.*, 
+      COUNT(comment_id) AS comment_count 
+      FROM reviews 
+      LEFT JOIN comments 
+      ON comments.review_id = reviews.review_id 
+      GROUP BY reviews.review_id 
+      ORDER BY created_at DESC;`
+    )
+    .then(({ rows: reviews }) => {
+      console.log(reviews);
+      return reviews;
+    })
+    .catch((err) => {
```

Comment:
you do not have a access to next inside the model, this should be handled only in your controllers :)